functions/tsconfig.json

```diff
@@ -7,8 +7,8 @@
     "sourceMap": true,
     "strict": true,
     "target": "es2017",
-    "esModuleInterop": true,
+    "esModuleInterop": true
```

Comment:
is this a prettier thing?

Reply:
probably, i just ran prettier on everything